src/app/src/components/SurveyDialogNotification.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import Typography from '@material-ui/core/Typography';
+import constant from 'lodash/constant';
+import attempt from 'lodash/attempt';
+import wrap from 'lodash/wrap';
+import partialRight from 'lodash/partialRight';
+import isError from 'lodash/isError';
+import moment from 'moment';
+
+const surveyURL = 'http://bit.ly/OARSurvey';
+
+const surveyDialogStyles = Object.freeze({
+    containerStyles: Object.freeze({
+        padding: '10px',
+    }),
+    titleStyles: Object.freeze({
+        marginLeft: '10px',
+    }),
+    contentStyles: Object.freeze({
+        fontSize: '20px',
+        margin: '5px 10px 10px 10px',
+    }),
+});
+
+const InvisibleDiv = constant(<div style={{ display: 'none ' }} />);
+
+const surveyEndDate = moment('2019-11-10');
+const currentDate = moment();
+const SURVEY_DIALOG_HAS_BEEN_DISPLAYED = 'SURVEY_DIALOG_HAS_BEEN_DISPLAYED';
+
+const setEntryToLocalStorage = partialRight(
+    wrap,
+    entry => window.localStorage.setItem(...[entry, entry]),
+);
+
+const trySetDialogHasBeenDisplayedToLocalStorage = attempt(
+    setEntryToLocalStorage,
+    SURVEY_DIALOG_HAS_BEEN_DISPLAYED,
+);
```

Comment:
This is clever, but I find the combination of `partialRight` and `wrap` a little opaque. Given that we are just setting a single value in a key-value store, consider this shorter version.

```diff
diff --git a/src/app/src/components/SurveyDialogNotification.jsx b/src/app/src/components/SurveyDialogNotification.jsx
index ab37a06..e5ec4cf 100644
--- a/src/app/src/components/SurveyDialogNotification.jsx
+++ b/src/app/src/components/SurveyDialogNotification.jsx
@@ -7,8 +7,6 @@ import DialogActions from '@material-ui/core/DialogActions';
 import Typography from '@material-ui/core/Typography';
 import constant from 'lodash/constant';
 import attempt from 'lodash/attempt';
-import wrap from 'lodash/wrap';
-import partialRight from 'lodash/partialRight';
 import isError from 'lodash/isError';
 import moment from 'moment';
 
@@ -33,14 +31,11 @@ const surveyEndDate = moment('2019-11-10');
 const currentDate = moment();
 const SURVEY_DIALOG_HAS_BEEN_DISPLAYED = 'SURVEY_DIALOG_HAS_BEEN_DISPLAYED';
 
-const setEntryToLocalStorage = partialRight(
-    wrap,
-    entry => window.localStorage.setItem(...[entry, entry]),
-);
-
-const trySetDialogHasBeenDisplayedToLocalStorage = attempt(
-    setEntryToLocalStorage,
-    SURVEY_DIALOG_HAS_BEEN_DISPLAYED,
+const trySetDialogHasBeenDisplayedToLocalStorage = () => attempt(
+    () => window.localStorage.setItem(
+        SURVEY_DIALOG_HAS_BEEN_DISPLAYED,
+        true,
+    ),
 );
 
 const dialogEntryIsNotSavedInLocalStorage = () => {
```

Reply:
Sounds good to me. I changed this in amended commit 782afe887b4e6dfe49182161eefaa868bd738775